Lib/profile/stack_collectors.py

```diff
@@ -0,0 +1,37 @@
+import collections
```

Comment:
Nit: I'm not thrilled about one module having a singular name (`collector` and `pstats_collector`) and this one using a plural name just because it's got two classes. The other ones can just as well grow more classes in the future, and we will not be renaming them. I'd just use a singular name here, too.